config/default.py

```diff
@@ -87,6 +87,36 @@
 NETCDF_CALENDAR_TYPE = "gregorian"
 NETCDF_UNLIMITED_DIMS = ['obs']
 NETCDF_CHUNKSIZES = 10000
+NETCDF_NONSCI_VARIABLES = [
+    # variables that shouldn't have a coordinates attribute added
+    # #10745 AC 2
+    'id',
+    'obs',
+    'deployment'
+]
+NETCDF_DEPTH_VARIABLES = [
```

Comment:
Should the current order of the entries be maintained as-is? If so you should add a comment stating that.

Reply:
will update comment to indicate that the order of the entries are important for determining a preferred variable in streams containing multiple matches

Reply:
done